netci.groovy

```diff
@@ -470,6 +414,66 @@ branchList.each { branchName ->
     }
 }
 
+// adds steps to a job to download coreclr artifacts and corefx artifacts and invoke run-test.sh.
```

Comment:
> Why is one worse than the other?

In my opinion, doing the runs in CoreCLR (using an LKG of CoreFX) is even worse.  `run-test.sh` makes it easy to replace the runtime in an already built drop of the tests, but we don't have something similar for Windows.  If we want to move to a `build.cmd`/`build.sh` workflow (which I am in favor of long term) we already have the technology to replace the CoreCLR when we deploy the tests (via `BUILDTOOLS_OVERRIDE_RUNTIME`).  We'll have to invent such technology (as well as technology to just run the tests without building the repository) or build corefx from source in the CoreFX runs.

Maybe we get some of this with the new dev workflow stuff, if so, that's be great. But right now we have basically no coverage for asserts in mscorlib and native code (especially cross plat globalization).  That feels like a large step backwards compared to what we had in TFS.